.gitignore

```diff
@@ -5,5 +5,7 @@ poetry.toml
 .ruff_cache/
 .vscode
 .env
+.env.test
 tests/assets/*.jsonl
 tests/assets/*.parquet
+humanloop
```

Comment:
do add a comment explaining why 'humanloop' and '.env.test' are needed - speaking of, why do we need both .env and .env.test